controller/src/test/java/io/pravega/controller/server/bucket/PravegaTablesStoreBucketServiceTest.java

```diff
@@ -282,7 +282,7 @@ public void testNodeCacheFailure() throws Exception {
     }
 
     @Test(timeout = 30000)
-    public void testSplitBrainScenrio() throws Exception {
+    public void testSplitBrainScenario() throws Exception {
```

Comment:
In my understanding, this test is checking that even if there is more than one bucket manager leader due to a network partition, in any case all the leaders would end up with the same distribution of buckets. Is my understanding correct @Bhupender-Y? I don't know if this test guarantees that this will always be the case, but maybe is good enough so far.

Reply:
Yes @RaulGracia, that's right . Also in this test we are ensuring that in case of two leader same bucket will not be running on two instances.